web_no_bubble/i18n/web_no_bubble.pot

```diff
@@ -0,0 +1,14 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0\n"
```

Comment:
s/Odoo Server 11.0/Odoo Server 12.0

Reply:
This is updated automatically from POT, so don't worry about this.